Crash when location_disabled_*.xml should be displayed.

I am experiencing a strange crash when location is disabled.  
It seems as if the new dimens resource is unavailable.
I can reproduce this isse when building from revision f6a39854afcd1463badea8f91f671787d3064f35 with AndroidStudio as well.

App version: 1.4 (F-Droid); version 1.3 is unaffected.
Device: Xiaomi Redmi 4X
ROM: Lineage 15.1 (Oreo), ResurrectionRemix 7.0.2 (Pie)
<details>
  <summary><b>Logcat</b></summary>

```
06-16 04:08:20.327  1439  4218 W ResourceType: For resource 0x7f050064, entry index(100) is beyond type entryCount(62)
06-16 04:08:20.327  1439  4218 W ResourceType: Failure getting entry for 0x7f050064 (t=4 e=100) (error -75)
06-16 04:08:20.327  1439  4216 W ResourceType: For resource 0x7f050065, entry index(101) is beyond type entryCount(62)
06-16 04:08:20.327  1439  4216 W ResourceType: Failure getting entry for 0x7f050065 (t=4 e=101) (error -75)
06-16 04:08:20.329  1439  1439 W ResourceType: For resource 0x7f050064, entry index(100) is beyond type entryCount(62)
06-16 04:08:20.329  1439  1439 W ResourceType: Failure getting entry for 0x7f050064 (t=4 e=100) (error -75)
06-16 04:08:20.330  1439  1439 E StatusBar: couldn't inflate view for notification com.dosse.airpods/0x1
06-16 04:08:20.330  1439  1439 E StatusBar: android.view.InflateException: Binary XML file line #2: Binary XML file line #2: You must supply a layout_height attribute.
06-16 04:08:20.330  1439  1439 E StatusBar: Caused by: java.lang.UnsupportedOperationException: Binary XML file line #2: You must supply a layout_height attribute.
06-16 04:08:20.330  1439  1439 E StatusBar: 	at android.content.res.TypedArray.getLayoutDimension(TypedArray.java:775)
06-16 04:08:20.330  1439  1439 E StatusBar: 	at android.view.ViewGroup$LayoutParams.setBaseAttributes(ViewGroup.java:7555)
06-16 04:08:20.330  1439  1439 E StatusBar: 	at android.view.ViewGroup$MarginLayoutParams.<init>(ViewGroup.java:7746)
06-16 04:08:20.330  1439  1439 E StatusBar: 	at android.widget.FrameLayout$LayoutParams.<init>(FrameLayout.java:439)
06-16 04:08:20.330  1439  1439 E StatusBar: 	at android.widget.FrameLayout.generateLayoutParams(FrameLayout.java:370)
06-16 04:08:20.330  1439  1439 E StatusBar: 	at android.widget.FrameLayout.generateLayoutParams(FrameLayout.java:368)
06-16 04:08:20.330  1439  1439 E StatusBar: 	at android.view.LayoutInflater.inflate(LayoutInflater.java:502)
06-16 04:08:20.330  1439  1439 E StatusBar: 	at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
06-16 04:08:20.330  1439  1439 E StatusBar: 	at android.widget.RemoteViews.inflateView(RemoteViews.java:3498)
06-16 04:08:20.330  1439  1439 E StatusBar: 	at android.widget.RemoteViews.-wrap1(Unknown Source:0)
06-16 04:08:20.330  1439  1439 E StatusBar: 	at android.widget.RemoteViews$AsyncApplyTask.doInBackground(RemoteViews.java:3600)
06-16 04:08:20.330  1439  1439 E StatusBar: 	at android.widget.RemoteViews$AsyncApplyTask.doInBackground(RemoteViews.java:3596)
06-16 04:08:20.330  1439  1439 E StatusBar: 	at android.os.AsyncTask$2.call(AsyncTask.java:333)
06-16 04:08:20.330  1439  1439 E StatusBar: 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
06-16 04:08:20.330  1439  1439 E StatusBar: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
06-16 04:08:20.330  1439  1439 E StatusBar: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
06-16 04:08:20.330  1439  1439 E StatusBar: 	at java.lang.Thread.run(Thread.java:764)
06-16 04:08:20.331  1439  1439 W StatusBar: removeNotification for unknown key: 0|com.dosse.airpods|1|null|10136
06-16 04:08:20.332  1264  2816 D NotificationService: onNotification error pkg=com.dosse.airpods tag=null id=1; will crashApplication(uid=10136, pid=4445)
06-16 04:08:20.334  4445  4445 D AndroidRuntime: Shutting down VM
06-16 04:08:20.335  4445  4445 E AndroidRuntime: FATAL EXCEPTION: main
06-16 04:08:20.335  4445  4445 E AndroidRuntime: Process: com.dosse.airpods, PID: 4445
06-16 04:08:20.335  4445  4445 E AndroidRuntime: android.app.RemoteServiceException: Bad notification posted from package com.dosse.airpods: Couldn't inflate contentViewsandroid.view.InflateException: Binary XML file line #2: Binary XML file line #2: You must supply a layout_height attribute.
```
</details>

> Here's a test APK without androidx. The layout will be all fucked up. If this version works, I'll fix it.
> [app-release.zip](https://github.com/adolfintel/OpenPods/files/4787414/app-release.zip)

This one still crashed as soon as i disable location, same error : /

```
06-16 17:20:23.861  1454  1454 E StatusBar: couldn't inflate view for notification com.dosse.airpods/0x1
06-16 17:20:23.861  1454  1454 E StatusBar: android.view.InflateException: Binary XML file line #2: Binary XML file line #2: You must supply a layout_height attribute.
06-16 17:20:23.861  1454  1454 E StatusBar: Caused by: java.lang.UnsupportedOperationException: Binary XML file line #2: You must supply a layout_height attribute.
```

--- a/OpenPods/app/src/main/res/layout/location_disabled_big.xml
+++ b/OpenPods/app/src/main/res/layout/location_disabled_big.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="@dimen/noti_location_big_height"
+    android:layout_height="120dp"
     android:background="@color/bgColorPrimary"
     android:gravity="center"
     android:padding="@dimen/padding_default">

--- a/OpenPods/app/src/main/res/layout/location_disabled_small.xml
+++ b/OpenPods/app/src/main/res/layout/location_disabled_small.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="@dimen/noti_location_small_height"
+    android:layout_height="70dp"
     android:background="@color/bgColorPrimary"
     android:gravity="center"
     android:padding="@dimen/padding_default">

--- a/OpenPods/app/src/main/res/values/dimens.xml
+++ b/OpenPods/app/src/main/res/values/dimens.xml
@@ -3,9 +3,6 @@
     <dimen name="padding_default" >12dp</dimen>
     <dimen name="padding_main"    >16dp</dimen>
 
-    <dimen name="noti_location_big_height"   >120dp</dimen>
-    <dimen name="noti_location_small_height" >70dp</dimen>
-
     <dimen name="noti_status_bat_size" >18dp</dimen>
 
     <dimen name="noti_status_small_item_width"  >80dp</dimen>

